core/controllers/admin.py

```diff
@@ -889,7 +903,7 @@ def put(self):
         self.render_json({})
 
 
-class ContributorUsersListHandler(base.BaseHandler):
+class ContributionRightsDataHandler(base.BaseHandler):
```

Comment:
Should this be `ContributionUsersHandler` to differentiate it from `ContributionRightsHandler`?

Reply:
Done.